### PR TITLE
Add room number - bug fix

### PR DESCRIPTION
**Pull request by LucasBywalec**

### CODE REVIEW

**Review by macszl — COMMENTED** [review]
LGTM

File: src/components/RegisterForm/RegisterForm.tsx [hunk 1]
```diff
@@ -62,7 +72,7 @@ export function RegisterForm() {
       validateOnChange={true}
       validateOnBlur={true}
       validate={validate}
-      initialValues={{ email: '', name: '', password: '', surname: '' }}
+      initialValues={{ email: '', name: '', password: '', roomNumber: 0, surname: '' }}
```
Comment (kaczor6418) [inline src/components/RegisterForm/RegisterForm.tsx:75]: `roomNumber: 0` I think with this approach you are going to get rid of input placeholder

**Review by kaczor6418 — CHANGES_REQUESTED** [review]

File: src/components/RegisterForm/RegisterForm.tsx (hunk 1 above)
Comment (LucasBywalec) [reply src/components/RegisterForm/RegisterForm.tsx]: It's hard to get rid of placeholder values complately as Formik requires some type of interface in <> brackets.

File: src/components/RegisterForm/RegisterForm.tsx [hunk 2]
```diff
@@ -47,6 +61,9 @@ export function RegisterForm() {
     if (values.name === undefined || values.name.length < minLength) {
       errors.name = t('registerForm.firstNameEmpty');
     }
+    if (values.roomNumber === undefined) {
+      errors.roomNumber = t('registerForm.roomNumberEmpty');
+    }
```
Comment (kaczor6418) [inline src/components/RegisterForm/RegisterForm.tsx:66]: If you add validation to `roomNumber` please validate vs correct room number 0+. Now someone can say my room is `dupa`

**Review by kaczor6418 — CHANGES_REQUESTED** [review]
ALmost there

**Review by kaczor6418 — CHANGES_REQUESTED** [review]
Register button is disabled, no errors shown:
Screenshot from 2023-05-16 15-04-09

**Review by kaczor6418 — CHANGES_REQUESTED** [review]
You need to fix commits history, I would recommend abandoning this PR and creating a new one with all commits that includes your change using cherry-pick

**Pull request closed without merging**